lib/ui/page/profile/follow/followingListPage.dart

```diff
@@ -5,6 +5,22 @@ import 'package:flutter_twitter_clone/ui/theme/theme.dart';
 import 'package:provider/provider.dart';
 
 class FollowingListPage extends StatelessWidget {
+   FollowingListPage({Key key, this.userList, this.profile}) : super(key: key);
+  final List<String> userList;
+  final UserModel profile;
+
+  static MaterialPageRoute getRoute(
+      {List<String> userList, UserModel profile}) {
+    return MaterialPageRoute(
+      builder: (BuildContext context) {
+        return FollowingListPage(
+          profile: profile,
+          userList: userList,
+        );
+      },
+    );
+  }
+
```

Comment:
@imsuriyan  You have created `getRoute` method and addded arguments to recieve user model and userlist but you forgot to pass these arguments to `UsersListPage`.  So remove `authState` variable and its uses from code and use new user list and  profile model. Check below code for refence.

``` dart
 @override
  Widget build(BuildContext context) {
    // var state = Provider.of<AuthState>(context);
    return UsersListPage(
        pageTitle: 'Following',
        userIdsList: userList,
        appBarIcon: AppIcon.follow,
        emptyScreenText:
            '${profile?.userName ?? profile.userName} isn\'t follow anyone',
        emptyScreenSubTileText: 'When they do they\'ll be listed here.');
  }
}
```